tests/ops/functions/test_eigvals.py

```diff
@@ -327,6 +327,21 @@ def test_composite_eigvals(self):
 
         assert np.allclose(mat_eigvals, op_eigvals)
 
+    @pytest.mark.parametrize("do_queue", [True, False])
+    def test_composite_do_queue_deprecation(self, do_queue):
+        """Test that CompositeOp raises a deprecation warning when do_queue is not None."""
+        factors = (qml.PauliX(0), qml.PauliY(0))
+        do_queue_deprecation_warning = (
+            "The do_queue keyword argument is deprecated. "
+            "Use qml.queuing.QueuingManager.stop_recording()"
+        )
+
+        with pytest.warns(UserWarning, match=do_queue_deprecation_warning):
+            qml.prod(*factors, do_queue=do_queue)
+
+        with pytest.warns(UserWarning, match=do_queue_deprecation_warning):
+            qml.sum(*factors, do_queue=do_queue)
+
```

Comment:
I think this change may be to the wrong test file.  This looks to be in `test_eigvals.py`. It should probably be in `ops/op_math/test_composite.py`.